apps/mobile/app.json

```diff
@@ -72,4 +72,4 @@
       "typedRoutes": true
     }
   }
-}
+}
```

Comment:
think your idea is missing prettier or editorconfig 
![image](https://github.com/user-attachments/assets/6d2888b7-727a-4dcf-9909-4f6b5294c0e9)

Reply:
Yeah I had `"editor.defaultFormatter": "vscode.json-language-features"` which was overriding for json i guess. Good catch.